src/components/Features/Output/DeployOutput.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const CONTRACT_URL = 'https://voyager.online/contract/';
+const TX_URL = 'https://voyager.online/tx/';
+const STRINGS = [
```

Comment:
maybe name this something like: `TXN_MSG_COMPONENTS` or something similar?

Reply:
how this is better?

Reply:
Not a big deal since it's anyway internal to this file.
I'm just thinking that 'STRINGS' is a very generic name, basically only hinting at the type.
Naming it according to its purpose looks better IMO.

but this shouldn't block.

Reply:
Gotcha. done